scripts/util.py

```diff
@@ -24,18 +24,17 @@ def oss_fuzz_one_target(
         fuzzout,
     ]
 
-    if dump is None:
-        job = subprocess.Popen(
-            cmd, stdout=subprocess.DEVNULL, stderr=subprocess.DEVNULL
-        )
-    else:
-        # todo: change mode to "a" if a fuzzer has multiple dump
-        with open(dump, "w") as dumpfile:
-            job = subprocess.Popen(
-                cmd,
-                stdout=dumpfile,
-                stderr=subprocess.DEVNULL,
-            )
+    if dump is not None:
+        cmd += [
+            "-e",
+            f"DUMP_FILE_NAME={dump}",
```

Comment:
This should be env var, not command line.

Reply:
`-e` in oss-fuzz's infra script is for setting env var in `base-runner` docker
```
> python3 oss-fuzz/infra/helper.py run_fuzzer -h
usage: helper.py run_fuzzer [-h] [--architecture {i386,x86_64,aarch64}] [--engine {libfuzzer,afl,honggfuzz,centipede,none,wycheproof}]
                            [--sanitizer {address,none,memory,undefined,thread,coverage,introspector,hwaddress}] [-e E] [--external] [--corpus-dir CORPUS_DIR]
                            project fuzzer_name [fuzzer_args ...]

positional arguments:
  project               name of the project or path (external)
  fuzzer_name           name of the fuzzer
  fuzzer_args           arguments to pass to the fuzzer

options:
  -h, --help            show this help message and exit
  --architecture {i386,x86_64,aarch64}
  --engine {libfuzzer,afl,honggfuzz,centipede,none,wycheproof}
  --sanitizer {address,none,memory,undefined,thread,coverage,introspector,hwaddress}
                        the default is "address"
  -e E                  set environment variable e.g. VAR=value
  --external            Is project external?
  --corpus-dir CORPUS_DIR
                        directory to store corpus for the fuzz target
```